webapp/src/components/card/subscription/index.tsx

```diff
@@ -17,83 +17,92 @@ type SubscriptionCardProps = {
-                <LabelValuePair
-                    labelIconClassName={'icon icon-account-outline icon-label'}
-                    value={`Subscription created by ${createdBy}`}
-                />
-                {
-                    (areaPath || repositoryName || targetBranch || pullRequestCreatedByName || pullRequestReviewersContainsName || pushedByName || mergeResultName || notificationTypeName) && (
-                        <div className='d-flex align-item-center margin-left-5'>
-                            <div className='card-filter'>
-                                <SVGWrapper
-                                    width={14}
-                                    height={14}
-                                    viewBox='0 0 12 12'
-                                >
-                                    {pluginConstants.SVGIcons.cardFilter}
-                                </SVGWrapper>
-                            </div>
-                            <div className='card-chip-wrapper'>
-                                {
+                        <p className={`ml-1 mb-0 font-bold color-${serviceType} text-capitalize`}>{serviceType}</p>
+                    </div>
+                    <div className='button-wrapper'>
+                        <IconButton
+                            tooltipText='Delete subscription'
+                            iconClassName='fa fa-trash-o'
+                            extraClass='delete-button'
+                            onClick={() => handleDeleteSubscrption(subscriptionDetails)}
+                        />
+                    </div>
+                </div>
+                <div className='project-details'>
+                    <LabelValuePair
+                        label={
+                            <SVGWrapper
+                                width={12}
+                                height={12}
+                                viewBox='0 0 10 10'
+                            >
+                                {pluginConstants.SVGIcons.workEvent}
+                            </SVGWrapper>
+                        }
+                        labelExtraClassName='margin-left-5'
+                        value={pluginConstants.common.eventTypeMap[eventType as EventType] ?? ''}
+                    />
+                    <LabelValuePair
+                        labelIconClassName={`icon ${channelType === mm_constants.PRIVATE_CHANNEL ? 'icon-lock-outline' : 'icon-globe'} icon-label`}
+                        value={channelName}
+                    />
+                    <LabelValuePair
+                        labelIconClassName={'icon icon-account-outline icon-label'}
+                        value={`Subscription created by ${createdBy}`}
+                    />
+                    {
+                        (areaPath || repositoryName || targetBranch || pullRequestCreatedByName || pullRequestReviewersContainsName || pushedByName || mergeResultName || notificationTypeName) && (
```

Comment:
```suggestion
                        const showFilter = areaPath || repositoryName || targetBranch || pullRequestCreatedByName || pullRequestReviewersContainsName || pushedByName || mergeResultName || notificationTypeName
```

create this `const` above `return` and use that here as it has become long now